src/Engine/ProtoCore/DSASM/Executive.cs

```diff
@@ -2837,7 +2839,8 @@ private void ExecuteDebug(int exeblock, int entry, List<Instruction> breakpoints
                     runtimeCore.DebugProps.DebugEntryPC = pc;
                 }
 
-                core.Rmem = rmem;
+                //runtimeCore.RuntimeMemory = rmem;
+                runtimeCore.RuntimeMemory = rmem;
```

Comment:
I agree, Ive added this investation to: http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6193
Although it shouldnt be changed this time, only the property name has changed here